db/migrate/20160203140743_change_column_previous_iteration_id_and_following_iteration_id_to_foreign_keys.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeColumnPreviousIterationIdAndFollowingIterationIdToForeignKeys < ActiveRecord::Migration
```

Comment:
As this code is executed as well, I would add the two magic comments at the beginning.